apisix/admin/consumers.lua

```diff
@@ -34,6 +38,40 @@ local function check_conf(username, conf, need_username, schema)
         if not ok then
             return nil, {error_msg = "invalid plugins configuration: " .. err}
         end
+
+        -- check duplicate key
+        for plugin_name, plugin_conf in pairs(conf.plugins or {}) do
+            local plugin_obj = plugin.get(plugin_name)
+            if not plugin_obj then
+                return nil, {error_msg = "unknown plugin " .. plugin_name}
+            end
+
+            if plugin_obj.type == "auth" then
+                local plugin_conf_copy = core.table.deepcopy(plugin_conf)
+                plugin.decrypt_conf(plugin_name, plugin_conf_copy, core.schema.TYPE_CONSUMER)
+
+                local key_field = utils.plugin_key_map[plugin_name]
+
+                if key_field then
+                    local key_value = plugin_conf_copy[key_field]
+
+                    if key_value then
+                        local consumer, _, err = consumer
+                            .find_consumer(plugin_name, key_field, key_value)
+                        if err then
+                            core.log.warn("failed to find consumer: ", err)
```

Comment:
```suggestion
                        local consumer = consumer.find_consumer(plugin_name, key_field, key_value)
                        if consumer then
                           ...
```

Reply:
The expected outcome is that it cannot be found, so I believe there is no need to record any logs.